server/test/unit/server/webservices/views/test_task_groups.py

```diff
@@ -0,0 +1,76 @@
+"""
+This module contains tests for the pulp.server.webservices.views.task_groups module.
+"""
+import mock
+
+from .base import assert_auth_READ
+from pulp.common.compat import unittest
+
+from pulp.server.exceptions import MissingResource
+from pulp.server.webservices.views.task_groups import TaskGroupSummaryView
+
+
+class TestTaskGroupSummary(unittest.TestCase):
+    """
+    Tests for TaskGroupSummaryView
+    """
+
+    @mock.patch('pulp.server.webservices.views.decorators._verify_auth',
+                new=assert_auth_READ())
+    @mock.patch('pulp.server.webservices.views.task_groups.TaskStatus.objects')
+    def test_get_task_group_summary_nonexistant(self, mock_objects):
+        """
+        Test get task_groups_summary with no tasks
+        """
+
+        mock_request = mock.MagicMock()
+        mock_objects.return_value.count.return_value = 0
+        task_group_summary = TaskGroupSummaryView()
+        self.assertRaises(MissingResource, task_group_summary.get, mock_request, 'mock_task')
```

Comment:
as an alternative you could use this structure to assert then the correctness of response code and body https://github.com/pulp/pulp/blob/master/server/test/unit/server/webservices/views/test_tasks.py#L181